content/blog/2022-05-05-newsletter.md

```diff
@@ -0,0 +1,190 @@
+---
+title: The XMPP Newsletter April 2022
+author: XMPP Communication Team and Contributors
+date: 2022-05-05
+categories: ['Newsletter']
+---
+
+Welcome to the XMPP Newsletter, great to have you here again! This issue covers the month of April 2022.
+
+Like this newsletter, many projects and their efforts in the XMPP community are a result of people’s voluntary work. If you are happy with the services and software you may be using, especially throughout the current situation, please consider saying thanks or help these projects! Interested in supporting the Newsletter team? Read more at the bottom.
+
+### Newsletter translations
+
+This is a community effort, and we would like to thank translators for their contributions. Volunteers are welcome! Translations of the XMPP Newsletter will be released here (with some delay):
+
+- French: [jabberfr.org](https://news.jabberfr.org/category/newsletter/) and [linuxfr.org](https://linuxfr.org/tags/xmpp/public)
+- German: [xmpp.org](https://xmpp.org/categories/newsletter/) and [anoxinon.de](https://anoxinon.de/blog/)
+- Italian: [nicfab.it](https://www.nicfab.it/)
+- Spanish: [xmpp.org](https://xmpp.org/categories/newsletter/)
+
+## XSF Announcements
+
+![XSF and Google Summer of Code 2022](/images/logos/GSoC_2022_Logo.png "XSF and Google Summer of Code 2022")
+
+- The XSF has been accepted as hosting organization at [Google Summer of Code 2022 (GSoC)](https://xmpp.org/community/gsoc-2022/).
+- XMPP Newsletter via mail: We migrated to our [own mail-list server](https://mail.jabber.org/mailman/listinfo/newsletter) and stopped using Tinyletter. Its read-only and you will receive the XMPP Newsletter on a monthly basis. It also eliminates the privacy concerns with Tinyletter.
+- By the way, have you checked our nice [XMPP RFC page](https://xmpp.org/rfcs/)? :-)
+
+## XSF fiscal hosting projects
+
+The XSF offers [fiscal hosting](https://xmpp.org/community/fiscalhost/) for XMPP projects. Please apply via [Open Collective](https://opencollective.com/xmpp). For more information, see the [announcement blog post](https://xmpp.org/2021/09/the-xsf-as-a-fiscal-host/). Current projects:
+
+- [Bifrost bridge: Offline messages exchange between Matrix and XMPP](https://opencollective.com/bifrost-mam)
+- [Collection Mellium Co-op projects](https://opencollective.com/mellium)
```

Comment:
This can just be "Mellium Co-op", I don't know why it got changed, but this sounds weird. It's just one entity managed by the XSF.